app/assets/javascripts/pools/pool_matches.js

```diff
@@ -0,0 +1,77 @@
+var hema = hema || {};
+hema.Pools = {};
+
+hema.Pools.PoolMatches = function(sortableMatchesSelector, saveButtonSelector,
+  alertSelector, matchSelector) {
+  this.$sortableMatches = $(sortableMatchesSelector);
+  this.$saveButton = $(saveButtonSelector);
+  this.matchSelector = matchSelector;
+  this.$alert = $(alertSelector);
+  this.savePath = this.$saveButton.data('path');
+  this.matchSaveClick = this.matchSaveClick.bind(this);
+  this.handleResponse =  this.handleResponse.bind(this);
+  this.handleError = this.handleError.bind(this);
+};
+
+hema.Pools.PoolMatches.prototype = {
+  init: function() {
+    this.sortable();
+    this.listen();
+  },
+
+  sortable: function() {
+    this.$sortableMatches.sortable();
+  },
+
+  listen: function() {
+    this.$saveButton.on('click', this.matchSaveClick);
+  },
+
+  matchSaveClick: function(event) {
+    event.preventDefault();
+
+    $.ajax({
```

Comment:
This should only happen if `this.newPayload` is different from `this.oldPayload`.